README.md

```diff
@@ -310,7 +477,7 @@ TODO Describe what your plugin does here
 
 ## Getting started
 
-TODO Tell users how to configure your plugin here, include screenshots, pipeline examples and 
+TODO Tell users how to configure your plugin here, include screenshots, pipeline examples and
```

Comment:
Do we want this text here?

Reply:
This is commented text from the README template used for Jenkins plugins. 

I'll remove it since it's no longer needed.